orc8r/cloud/go/obsidian/swagger/handlers/handlers_test.go

```diff
@@ -65,7 +65,7 @@ func Test_GenerateCombinedSpecHandler(t *testing.T) {
 	defer registry.RemoveServicesWithLabel(orc8r.SwaggerSpecLabel)
 
 	for i, s := range services {
-		registerServicer(t, s, tags[i])
+		registerServicer(t, s, tags[i], swagger_lib.TagDefinition{})
```

Comment:
Everywhere we're aliasing as `swagger_lib`, let's go ahead and convert to `spec` alias. You can add that change as part of this pr for expediency